ntbs-service/wwwroot/source/Components/AutocompleteSelect.ts

```diff
@@ -9,7 +9,6 @@ const AutocompleteSelect = Vue.extend({
         var options:any = {
             placeholder: this.$props.placeholder,
             autoselect: false,
-            preserveNullOptions: true,

```

Comment:
this is what caused the weird bit of whitespace (which was the blank value) - we can just as easily select blank value by pressing the delete "x" in the input field

Reply:
oh really? odd. Happy as long as there's an option to clear and save it as cleared.
there's an 'x' button? not seen it on any examples, it's also not showing up on INT?

Reply:
Oh that's actually just an IE feature - not present in Chrome. However, when you click on the field the current selection is automatically highlighted so easy to erase again.

Reply:
Yeah, I think that's fine - by turning this into an autocomplete we're basically telling users to use keyboards for the right value anyways!